src/Qir/Runtime/unittests/CMakeLists.txt

```diff
@@ -17,7 +17,7 @@ add_executable(qir-runtime-unittests
 )
 
 target_include_directories(qir-runtime-unittests PUBLIC 
-  "${PROJECT_SOURCE_DIR}/../Common/externals/catch2"
+  "${PROJECT_SOURCE_DIR}/../Common/Externals/catch2"
```

Comment:
Changing the casing of the folder like this will break anyone who is using Linux devcontainers, forcing them to manually rename the folder, which may not be obvious. Why is this change being made now? The paths in the CMakeLists.txt and paths in the repo agreed, so there was no bug here before. What motivated the change?

Reply:
We have 2 folders, 
qsharp-runtime/src/Qir/Common/**e**xternals (starts with lower case `e`),
qsharp-runtime/src/Qir/Tools/**E**xternals (starts with upper cased `E`).
Both need to be excluded from clang-formatting. It is more efficient to exclude both with a [single line](https://github.com/microsoft/qsharp-runtime/blob/10c22457fc27dd05206f9746b7fbc1e33ef8b07d/src/Qir/check-sources-formatted.ps1#L23). But on Linux the paths are case-sensitive. So the single line does not exclude both dirs. 
To exclude both with a single line I unified those two directory names.

Reply:
Can you please show what is about to be broken with this change?

Reply:
Actually, never mind, I tried this locally and it was odd but didn't cause a problem or break anything. While the rename gets ignored on Windows, which can lead to some weird corner cases, in this case you avoid the break by updating everywhere at the same time. I've gotten overly suspicious of capitalization changes after getting burned by them before, because git on Windows ignores this rename (my local repo when merging in this branch keeps the lowercase "external" folder name) and that can lead to confusion in the future when it looks like paths checked into the repo are wrong. But for Linux devcontainers, the source code is shared from a Windows volume, meaning even then the tools will find the right folder because despite being case  sensitive the files are being served into Linux by the case insensitive Windows file system. Anyone in this mixed mode who thinks they need to update the file paths in the CMakeLists.txt will hit build failures, so that should protect other users, and folks who develop directly in Linux will get the right names from case sensitive git behavior.

Sorry for the confusion, I was wrong to block for this!